### PR TITLE
#171 implement pagination

### PR DESCRIPTION
**Pull request by Lvyshnevska**
Pagination for Companies list was implemented on the back-end. Profile tests were updated according to new response data (taking into account pagination functionality). Besides, imports order were fixed in some files.

### CODE REVIEW

File: forum/settings.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 """
 
 from datetime import timedelta
+from os import nice
```
Comment (ProDG) [inline forum/settings.py:14]: Is it used?

File: profiles/migrations/0005_alter_profile_options_alter_savedcompany_options_and_more.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='profile',
+            options={'ordering': ['profile_id']},
+        ),
+        migrations.AlterModelOptions(
+            name='savedcompany',
+            options={'ordering': ['company_id']},
```
Comment (ProDG) [inline profiles/migrations/0005_alter_profile_options_alter_savedcompany_options_and_more.py:19]: I really, really discourage you of using `Meta.ordering` on models.

While it works nicely in cases when project is small and there is not a lot of records in DB, it brings just HUGE amount of issues on bigger projects.

Ordering must be applied explicitly, _when_ needed and _where_ needed.

Comment (ProDG) [reply profiles/migrations/0005_alter_profile_options_alter_savedcompany_options_and_more.py:19]: To give you more context. This ordering will be applied on each DB request to the model, doesn't matter if there is a sense to do ordering (which, by the way, is not 'for free', it's pretty heavy operation on DB level).

I'm not 100% sure, but it's possible that with `Meta.ordering` on model you'll have issues trying to implement ordering in DRF filters etc.

So, just don't use this.

File: forum/settings.py (hunk 1 above)
Comment (Lvyshnevska) [reply forum/settings.py:14]: o_O It's crazy VS Code. I'll remove it.

**Review by ProDG — APPROVED** [review]

**Review by GeorgiyII — APPROVED** [review]

**Pull request merged**